README.md

```diff
@@ -194,9 +211,6 @@ symlogpdf(m2).evalf(3)
                            ‾‾‾                                     ‾‾‾          
                          _j1 = 1                                 _j1 = 1        
 
```

Comment:
This looks ugly in the Readme. I'm not sure why, since the source seems fine.

Reply:
Good catch! I think this is a quirk of `Weave.jl`  depending on the width of the terminal it's run from. There really should be a way to make that not matter, but for now I'll just re-run it